Next F1 Race Schedule.js

```diff
@@ -8,7 +8,7 @@
 // --------------------------------------------------
 // Constants & Setup - DO NOT EDIT
 // --------------------------------------------------
-const SCRIPT_VERSION = "4.8";
+const SCRIPT_VERSION = "4.8.1";
```

Comment:
```suggestion
const SCRIPT_VERSION = "4.9";
```
Version check gonna fail with semver format